jib-core/src/test/resources/core/json/v21manifest.json

```diff
@@ -5,7 +5,7 @@
     {"blobSum":"sha256:5bd451067f9ab05e97cda8476c82f86d9b69c2dffb60a8ad2fe3723942544ab3"}
   ],
   "history": [
-    {"v1Compatibility":"some v1-compatible object"},
+    {"v1Compatibility": "{\"architecture\":\"amd64\",\"config\":{\"Env\":[\"JAVA_HOME=/opt/openjdk\",\"PATH=/opt/openjdk/bin\"],\"Entrypoint\":[\"/opt/openjdk/bin/java\"],\"Cmd\":[\"-version\"]},\"created\":\"2019-04-17T14:02:34.79404123Z\"}"},
```

Comment:
Would is be possible to create a new test with a this container-configuration object?  The previous test wasn't wrong: the `v1Compatibility` could be some arbitrary object.

Reply:
The next element is `"another v1-compatible object"`, so at least we know the method doesn't fail with an arbitrary string. I just didn't like adding the `@VisibleForTesting` method `getV1Compatibility()` used only in testing and nowhere else.